src/utils.ts

```diff
@@ -166,3 +172,37 @@ export function getProvider(chainId: ChainId): ethers.providers.BaseProvider {
 
   return ethers.getDefaultProvider(ethers.providers.getNetwork(chainId));
 }
+
+export async function validateAmount(amount: number, maxAvalible: number): Promise<void> {
```

Comment:
the function name and placement is generic, but the logic is not. why do you assume all number validation wants the value to be above 0? why would the error message talk about magic if this is not the only token we support in Automate? either rename and move the function appropriately or make it actually generic

Reply:
There is  an issue kinda related with this actually. I moved this function in side the file but validation is not working properly for amount field.
if i use <Input.Group compact> inside   <Form.Item> tag the validation while typing is not working. 
i think its mentioned here [](https://github.com/ant-design/ant-design/issues/5790) but i couldn't solve it. 

![image](https://user-images.githubusercontent.com/34207598/176639784-fc9b645a-febb-48ac-8c22-8281b1d7b455.png)